README.md

```diff
@@ -911,6 +916,13 @@ Other Style Guides
       // ...
     }
 
+    // best
+    function foo(data) {
+      // This prevents the developer from confusing the order of the arguments.
```

Comment:
but it also defies convention, required arguments must always be positional. Options objects are only for optional arguments.